integration_test/gce/gce_testing.go

```diff
@@ -295,6 +295,7 @@ type VM struct {
 	Project     string
 	Network     string
 	Platform    string
+	ImageSpec   string
```

Comment:
this should be documented too. So should `Platform` but that's outside the scope of this PR. Everything else is self-explanatory (or already has a comment).

Reply:
Fixed